src/runtime/parallel/README.md

```diff
@@ -1,11 +1,51 @@
 Parallelization is achieved by having multiple child processes running scenarios.
 
+#### Customizable work assignment
+Cucumber exposes customization of worker assignment via `setParallelCanAssign`.
+The example below overrides the default, `() => true` which processes test cases 
+indiscriminately, with a 1, 2 skip a few processing scheme. This means, the first
+2 *remaining* steps will be executed then the 4th *remaining* step will be executed.
+
+```typescript
+import { setParallelCanAssign } from '@cucumber/cucumber'
+const counter = 0;
+setParallelCanAssign((pickleInQuestion, picklesInProgress) => counter++ % 5 < 1)
+/** Processing order of: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+ * After   Worked | Waiting    
+ * first 2 [0, 1] | [2, 3, 4, 5, 6, 7, 8, 9]
+ * 4th     [0, 1, 5] | [2, 3, 4, 6, 7, 8, 9]
+ * first 2 [0, 1, 5, 2, 3] | [4, 6, 7, 8, 9]
+ * 4th     [0, 1, 5, 2, 3, 8] | [4, 6, 7, 9]
+ * fisrt 2 [0, 1, 5, 2, 3, 8, 4, 6] | [7, 9]
+ * 4th     [0, 1, 5, 2, 3, 8, 4, 6, 9] | [7]
+ * first    [0, 1, 5, 2, 3, 8, 4, 6, 9, 7]
+ */ 
+```
```

Comment:
This example seems very complex to me. Thoughts on trying to make the example something like "will not run a pickle with the word 'complex' in the name if there is a running pickle with the word 'complex' in the name

Reply:
I think using names or tags in an example would be good as that I how I would anticipate users actually using this.

Reply:
Sure thing. Will update accordingly.